connector/routingconnector/router.go

```diff
@@ -109,6 +111,16 @@ func (r *router[C]) registerDefaultConsumer(pipelineIDs []pipeline.ID) error {
 	return nil
 }
 
+// convert conditions to statements
+func (r *router[C]) normalizeConditions() {
+	for i := range r.table {
+		item := &r.table[i]
+		if item.Condition != "" {
+			item.Statement = fmt.Sprintf("route() where %s", item.Condition)
+		}
```

Comment:
We repeat the same process in `registerRouteConsumers` https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/3f9f88b8316345dd0cfc535ae1543e8412899cea/connector/routingconnector/router.go#L150-L154
Probably we can remove this block.

Reply:
Sure, I deleted `getStatementFrom` altogether as it wasn't doing anything necessary.